### DISCUSSION TITLE
What are Openthread Network simulator with visualization real life parameters?

### DISCUSSION THREAD

**Discussion by MartinsMihaeljans** [openthread/openthread#9260] [Q&A] [answered]
Hello,

I made a simulation of 3 network devices arranging them with cli like this:
add router x 400 y 400 rr 125
add fed x 300 y 400 rr 125
add fed x 500 y 400 rr 125

I understand that x y are distance in display points from upper left corner of browser window, and rr is radio range also in display points.

**What could be a real life equivalent of display points?**

**Comment by jwhui** [comment, marked as answer]
The coordinates and range are not tied to any real-world physical units. You can decide what a unit in OTNS coordinates maps to in the physical world.

**Reply by MartinsMihaeljans** [reply]
Thank you @jwhui. I was simply unsure as the simulator does have some radio tx and rx parameters and dB units, so I assumed that the distances might be somewhat real as well.

**Reply by jwhui** [reply]
The OTNS radio model is currently a very simple disk model where radius is specified by `rr`.
